manim/animation/creation.py

```diff
@@ -192,10 +192,8 @@ def construct(self):
 
     def __init__(
         self,
-        mobject: typing.Union[VMobject, OpenGLVMobject],
-        rate_func: typing.Callable[[float, float], np.ndarray] = lambda t: smooth(
-            1 - t
-        ),
+        mobject: Union[VMobject, OpenGLVMobject],
+        rate_func: Callable[[float], float] = lambda t: smooth(1 - t),
```

Comment:
Why is it `Callable[[float], float]` and not only `Callable[float]`

Reply:
Ah, I see:
https://mypy.readthedocs.io/en/stable/cheat_sheet_py3.html#functions